weblate/addons/autotranslate.py

```diff
@@ -34,7 +34,7 @@ class AutoTranslateAddon(BaseAddon):
     multiple = True
     icon = "language.svg"
 
-    def component_update(self, component: Component) -> None:
+    def component_update(self, component: Component, **kwargs) -> None:
```

Comment:
I don't really like using `**kwargs` here; it makes it easier to break things when the API changes, so use the new arg here directly.